hints/products-of-array-discluding-self.md

```diff
@@ -0,0 +1,39 @@
+<br>
+<details class="hint-accordion">  
+    <summary>Recommended Time & Space Complexity</summary>
+    <p>
+    You should aim for a solution with <code>O(n)</code> time and <code>O(1)</code> space, where <code>n</code> is the size of the input array.
```

Comment:
For this one i think it's fine to have O(n) space since it's only a medium

Reply:
I thought it would be better to have a solution from your video, but fine, now I understood what you are trying to say. I will be more beginner friendly.